### PR TITLE
GamePack licensing

### PR DESCRIPTION
**Pull request by illwieckz**
use BSD 3 clauses for gamepack too

### CODE REVIEW

File: src/entities/header.txt [hunk 1]
```diff
@@ -2,15 +2,40 @@ THIS FILE IS AUTOGENERATED, DO NOT MODIFY
...
+  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 
 Contributors:
-  1999 inolen, Suicide
+  2014 neumond, Viech
+
+Greetings:
+  We can thank these authors for the old Tremulous GPP's entities.def that was
+  used as documentation for that complete rewrite:
+  1999 inolen, Suicide 20
```
Comment (mjstahlberg) [inline src/entities/header.txt]: "20"?

File: src/entities/header.txt [hunk 2]
```diff
@@ -2,15 +2,40 @@ THIS FILE IS AUTOGENERATED, DO NOT MODIFY
...
+  ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 
 Contributors:
-  1999 inolen, Suicide
+  2014 neumond, Viech
+
+Greetings:
+  We can thank these authors for the old Tremulous GPP's entities.def that was
+  used as documentation for that complete rewrite:
```
Comment (mjstahlberg) [inline src/entities/header.txt]: Better: "We thank these authors for contributing to Tremulous' entity definition file that was used as documentation when creating this one."

Comment (illwieckz) [reply src/entities/header.txt]: fixed

File: src/entities/header.txt (hunk 1 above)
Comment (illwieckz) [reply src/entities/header.txt]: “Suicide 20”, including space an number what the nickname of the guy:
https://github.com/ppetr/tremulous-gpp-mods/blob/master/misc/entities.def#L5-L7

**Pull request merged**